pkg/trace/api/otlp.go

```diff
@@ -338,6 +338,10 @@ func convertSpan(rattr map[string]string, lib pdata.InstrumentationLibrary, in p
 		name = "opentelemetry." + name
 	}
 	traceID := in.TraceID().Bytes()
+	meta := map[string]string{}
```

Comment:
We already know it will have at least as many elements as `rattr`, and then some, so maybe:
```go
meta := make(map[string]string, len(rattr))
```